server/sonar-server/src/main/java/org/sonar/server/edition/StandaloneEditionManagementStateImpl.java

```diff
@@ -159,6 +160,18 @@ public synchronized PendingStatus finalizeInstallation() {
     return this.pendingInstallationStatus;
   }
 
+  @Override
+  public synchronized PendingStatus uninstall() {
+    ensureStarted();
+    changeStatusToFrom(UNINSTALL_IN_PROGRESS, UNINSTALL_IN_PROGRESS, NONE);
```

Comment:
I don't think the state should allow uninstall to be called twice. Only the WS need to support it.

Reply:
I won't change this at this point, we can change it later if anything